Formula/qemu.rb

```diff
@@ -42,6 +33,12 @@ class Qemu < Formula
   depends_on "snappy"
   depends_on "vde"
 
+  on_linux do
+    depends_on "gcc" => :build
```

Comment:
We should check that this doesn't have a runtime dependency on `gcc`. See https://github.com/Homebrew/homebrew-core/pull/83628#discussion_r693326619